src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheckTest.java

```diff
@@ -100,7 +98,7 @@ public void testHtmlDisallowed() throws Exception {
         final String[] expected = {
             "1: " + getCheckMessage(MSG_PACKAGE_INFO),
         };
-        verify(checkConfig,
+        verifyWithInlineConfigParser(
```

Comment:
All test methods need to be updated like this

Reply:
@romani please refer https://github.com/checkstyle/checkstyle/pull/12322#discussion_r1002746153

Reply:
@romani some of the verify() methods have more than two parameters, is it possible to use verifyWithInlineConfigParser() for them? as verifyWithInlineConfigParser() only uses single parameter for path of input file.

Reply:
You do not need all other parameters

Reply:
Look at other test classes for example of usage